components/General/TWModal/index.tsx

```diff
@@ -21,7 +21,7 @@ export const TWModal: React.FC<TWModalProps> = ({ open, onClose, size = 'default
     return (
         <Transition.Root show={open} as={Fragment}>
             <Dialog as="div" className="fixed z-10 inset-0 overflow-y-auto" onClose={() => onClose()}>
-                <div className="flex items-end justify-center min-h-screen pt-4 px-4 pb-20 text-center sm:block sm:p-0">
+                <div className="flex items-end justify-center min-h-screen pt-4 pb-20 text-center sm:block sm:p-0">
```

Comment:
do you mind adding a min-width 611px to this

Reply:
Added a styled component with a media query min-width: 611px

I wanted to do the same for Dialog.Overlay but I am having a hard time with typescript, styled components and headlessUI compound component structure. TS is throwing some crazy errors when passing props.

Type '{ isOverlay: boolean; }' is not assignable to type 'IntrinsicAttributes & Props<"div", OverlayRenderPropArg, OverlayPropsWeControl>'.